plaso/storage/interface.py

```diff
@@ -122,23 +271,40 @@ def GetEventTagByIdentifier(self, identifier):
 
     Returns:
       EventTag: event tag or None if not available.
+
+    Raises:
+      OSError: if an invalid identifier is provided.
+      IOError: if an invalid identifier is provided.
     """
+    return self._GetAttributeContainerByIdentifier(
+        self._CONTAINER_TYPE_EVENT_TAG, identifier)
 
-  @abc.abstractmethod
+  # pylint: disable=redundant-yields-doc
```

Comment:
Remove override, this method should return a generator

Reply:
Done